README.md

```diff
@@ -1,6 +1,148 @@
-# Regolamento Fantainformatica
-
-### Amministratori
+# Regolamento fantainformatica 2021-22
 
 - Presidente: Andrea `@andrea9lli`
-- Vice-presidente: Marco `@mlsmrc`
\ No newline at end of file
+- Vice-presidente: Marco `@mlsmrc`
+
+## Modalità CLASSIC
+
+### Regole mercato iniziale
+* Minimo squadre: **8**
+* Budget iniziale: **500** crediti
+* Base d'asta: **1**
+* Modalità d'asta: Asta a chiamata libera per ruolo (P -> D -> C -> A)
+* Una sessione di asta al termine della sessione di calciomercato estivo della Serie A (di solito 31 agosto). La data è da concordare
+* Verrà tenuta aperta una sessione di mercato solo scambi tra utenti fino all'inizio della successiva giornata
+
+### Regole mercato di riparazione
+* Una sola sessione di asta di riparazione al termine del calciomercato della Serie A (di solito 31 gennaio). La data è da concordare
+* Sessione mercato solo scambi attivo all'interno dell'app dal 1 gennaio
+* Verrà effettuata una riassegnazione dei crediti in ordine crescente e in base alla posizione in classifica dei giocatori al termine del girone di andata, così come segue:
+	* 1° posto: **100**
+	* 2° posto: **110**
+	* 3° posto: **120**
+	* 4° posto: **130**
+	* ...
+	* 8° posto: **170**
+
+* Numero **illimitato** di svincoli e acquisti
+* In caso di svincoli, l'utente riprenderà il:
+	* **100%** dei crediti in caso il giocatore vada in un altro campionato
+	* **50% -1**   dei crediti altrimenti (tranne per i giocatori pagati **1** credito, per i quali verrà restituito **1**)
```

Comment:
perchè 50 -1?

Reply:
Sempre stato così negli anni precedenti. Mi sono reso conto che ridare 1 a chi svincola giocatori precedentemente acquistati a 1 credito ha senso solo nel caso in cui NON venissero riassegnati i crediti, rendendo quindi impossibile eventuali cambi. Quindi potremmo/dovremmo trovare un'alternativa

Reply:
ma è come una tassa sullo svincolo. non ne vedo l'utilità cmq ok